Dockerfile.test

```diff
@@ -1,8 +1,9 @@
-FROM anem-per-feina:local
+FROM geekscat/anem-per-feina:latest
```

Comment:
maybe it's a good idea to keep pointing to the local image build, is the way to ensure that if something changes on local Dockerfile, is also processed into the tests (i.e new dependencies for this new branch) 

I mean, while testing in local you should expect to test it vs local build, not vs latest pubished tag

Reply:
This will download the image from dockerhub but in the dockerfile we are installing the requirements on top and in the docker-compose we have the volumes mapping the code that you have in your local. Then you are testing what you have in your local. I tried many times and works.
But anyway, I remove it. I'm building an image from scratch. I cannot pull the local image in the CI. I should push it in the registry and then pull it. I think it's better to rebuilding it from scratch since we are already doing it, the only thing that I did is add the COPY

Reply:
This will download the image from dockerhub but in the dockerfile we are installing the requirements on top and in the docker-compose we have the volumes mapping the code that you have in your local. Then you are testing what you have in your local. I tried many times and works.
But anyway, I remove it. I'm building an image from scratch. I cannot pull the local image in the CI. I should push it in the registry and then pull it. I think it's better to rebuilding it from scratch since we are already doing it, the only thing that I did is add the COPY

Reply:
Sure, but there are more types of changes that can be introduced (like base system installations), or base layer updates (change FROM def), new layers, or any other changes to the original Dockerfile

Reply:
Yes we should do a base image to install the wait for it for example but this is out of the scope of this ticket IMO